src/components/ContactsList/ContactsListOrSpinner.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react'
+import { PropTypes } from 'prop-types'
+
+import ContactsList from '.'
+import SpinnerContact from '../Components/Spinner'
+
+class ContactsListOrSpinner extends Component {
```

Comment:
It could go in `ContactsList` or make this component more abstract to be able to use it with any data that could be fetched in multiple steps (name it `Pagination`? `DataLoader`?)

Reply:
Created a ContactsListDataLoader. I don't want to see this behavior generalized.